packages/webdriverio/src/context.ts

```diff
@@ -19,6 +19,7 @@ export function getContextManager(browser: WebdriverIO.Browser) {
 export class ContextManager {
```

Comment:
@christian-bromann 

There were also no UT's for this class. Was that on purpose?

Reply:
Nope, we have some decent e2e tests for this which is why I skipped adding unit tests as I wasn't clear how the structure of this class will change.